src/gui/tray/UserModel.cpp

```diff
@@ -417,12 +417,16 @@ ActivityListModel *User::getActivityModel()
 
 void User::openLocalFolder()
 {
+    const auto folder = getFolder();
+
+    if (folder != nullptr) {
 #ifdef Q_OS_WIN
-    QString path = "file:///" + this->getFolder()->path();
+        QString path = "file:///" + folder->path();
```

Comment:
Not introduced by you but since we're looking at that horrible ifdef and string concatenation... what about switching to QUrl::fromLocalFile()?

This way we wouldn't rely on the QString -> QUrl implicit conversion (we should really disable this...) during the openUrl call.